governance/votes/Executive Vote - November 15, 2023.md

```diff
@@ -0,0 +1,237 @@
+	- [Borrow Cap](https://docs.sparkprotocol.io/developers/features/supply-borrow-caps#borrow-caps): **2,000 WBTC**
+	- [Reserve Factor](https://docs.aave.com/risk/asset-risk/risk-parameters#reserve-factor): **20%**
+	- Base rate: **0%**
+	- Optimal rate: **2%**
+	- Max rate: **302%**
+	- Optimal utilization: **60%**
+	- [Efficiency mode or eMode](https://docs.sparkprotocol.io/developers/features/efficiency-mode-emode): **No**
+	- [Isolation Mode](https://docs.sparkprotocol.io/developers/features/isolation-mode): **No**
+	- Debt ceiling: n/a
+		- ~$78,000,000 effective risk exposure (~10% of TVL, ~155% of Maker surplus)
+- Increase the Spark Lend Ethereum rETH and wstETH [supply caps](https://docs.sparkprotocol.io/developers/sparklend/features/supply-borrow-caps#supply-caps):
+	- Increase the rETH supply cap from 60,000 rETH to **80,000 rETH**.
+	- Increase the wstETH supply cap from 400,000 wstETH to **800,000 wstETH**.
+- Adjust the Spark Lend Ethereum **and** Gnosis Chain ETH [interest rate model](https://docs.aave.com/risk/liquidity-risk/borrow-interest-rate#interest-rate-model) parameters:
+	- Base rate: **0%**
+	- Optimal Rate: **3.2%**
+	- Optimal Utilization: **90%**
+	- Maximum Rate: **123.2%**
+
+### Adjust Spark Lend D3M Maximum Debt Ceiling (`line`)
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmVbrypf#poll-detail)\
+*Proposal*: [Forum Post](https://forum.makerdao.com/t/proposal-to-adjust-sparklend-parameters/22542)
+
+If this executive proposal passes, the following change will be made to the Spark Lend Direct Deposit Module Maximum Debt Ceiling:
+
+- Increase the DIRECT-SPARK-DAI [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) from 400 million DAI to **800 million DAI**.
+
+### Launch Project Funds Transfer
+
+*Authorization*: [Forum Notification](https://forum.makerdao.com/t/utilization-of-the-launch-project-under-the-accessibility-scope/21468/6)
+
+If this executive proposal passes, the following transfers will be made to the Launch Project as part of the funding defined in the [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108#9-launch-project):
+
+- Transfer **2,200,000 DAI** to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+- Transfer **500.00 MKR** to [0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F](https://etherscan.io/address/0x3C5142F28567E6a0F172fd0BaaF1f2847f49D02F).
+
+### DAO Resolutions Pertaining to HVBank (RWA009-A)
+
+*Authorization*: [Governance Poll](https://vote.makerdao.com/polling/QmNgKzcG)\
+*Proposal*: [Forum Post](https://forum.makerdao.com/t/huntingdon-valley-bank-transaction-documents-on-permaweb/16264/19)
+
+If this executive proposal passes, two DAO Resolutions pertaining to the HVBank facility will be approved. These are broken down as follows:
```

Comment:
```suggestion
If this executive proposal passes, a DAO Resolution and Standing Instructions pertaining to the HVBank facility will be approved. These are broken down as follows:
```

Reply:
I would use the same language Ecosystem used

Reply:
see: https://github.com/0xldr/community/commit/0cdf553554a882f52c36fc8d4583849797a13265